### PR TITLE
entity: Add Fill function + fix

### PR DESCRIPTION
**Pull request by fern4lvarez**
This should replace #4

### CODE REVIEW

File: entity.go [hunk 1]
```diff
@@ -34,7 +32,7 @@ func (e *Entity) Draw(s *Screen) {
 	}
 }
 
-func (e *Entity) Tick(ev termbox.Event) {}
+func (e *Entity) Tick(ev Event) {}
```
Comment (JoelOtter) [inline entity.go]: Good spot!

File: entity.go [hunk 2]
```diff
@@ -57,3 +55,13 @@ func (e *Entity) SetPosition(x, y int) {
 func (e *Entity) SetCell(x, y int, c *Cell) {
 	renderCell(&e.canvas[x][y], c)
 }
+
+// Fill fills the canvas of the Entity with
+// a c Cell.
```
Comment (JoelOtter) [inline entity.go]: This should be

```go
// a Cell c.
```

(Sorry to be picky but I think we need to keep the GoDoc as perfect as possible!)

Comment (fern4lvarez) [inline entity.go]: :+1: 
Actually, if you want to rephrase the whole func description, just go ahead and put it here - I'll never be a native english speaker :smile:

Comment (JoelOtter) [inline entity.go]: You had me fooled. ;)

**Comment by fern4lvarez** [comment]
bump!

**Comment by JoelOtter** [comment]
Hey dude - unless I'm very much mistaken I'm still waiting for that change
to the comment! :)

**Comment by fern4lvarez** [comment]
Check the last commit (I push -f'ed)

**Comment by JoelOtter** [comment]
So you did...GitHub decided not to inform me. Thanks for this! :+1:

**Pull request merged**